requirements.txt

```diff
@@ -2,5 +2,9 @@
 pytest
 pytest-cov
 pytest-asyncio
+pytest-ordering
```

Comment:
Why is this module necessary? by design tests should not depend on the state of other tests, therefore the order in which they are executed is irrelevant

Reply:
by design, they should not, but I want to make sure no other test executes while I mess with the connection.